Update index.js

Razielini opened the pull request:
Solved

Razielini committed:
Update index.js

Solved

Razielini committed:
Update index.js

Improved

hyfi06 commented on src/index.js:
```diff
@@ -1,6 +1,19 @@
+    let interaction = (size - 1) / 2
+    while(interaction >= 0) {
+      resultInvert = backupResult + resultInvert
+      backupResult = `${' '.repeat(interaction)} ${'*'.repeat((size - (interaction * 2)))} \n`
```
Si usas templates literals cada espacio cuenta, por lo que aquí estás añadiendo un espacio de más. `)} ${'*` corre los test antes de enviar tu respuesta.

hyfi06 reviewed (changes requested):
the challenge wasn't solved

hyfi06 commented:
No pasan los test. corre `npm test` antes de enviar tu solución.

Razielini committed:
Update index.js

Again, lo probe antes y el resultado fue bueno, por si acaso ahora reinicie el IDE
